src/eme.js

```diff
@@ -291,8 +291,10 @@ export const defaultGetLicense = (keySystemOptions) => (emeOptions, keyMessage,
     }
 
     if (response.statusCode >= 400 && response.statusCode <= 599) {
+      const cause = String.fromCharCode.apply(null, new Uint8Array(responseBody));
```

Comment:
should we try to JSON.parse this to return it as an object if it is JSON?

Reply:
I lean towards no, because we don't want to return a string or an object here. Perhaps we want to have multiple properties like `raw` and `json`?

Reply:
ultimately I think we want to return a string for `cause`.

Reply:
Yeah, let's just keep it as a string and not do any extra work here.

Reply:
I've seen errors before returned as not `arraybuffer`s. Do we want to consider those (even if they go against the request's `responsetype`)?

Reply:
oh, we could check the content type of the response and handle it accordingly. But probably best to just convert from arraybuffer to string/whatever and be done with it.